cnvlib/gary.py

```diff
@@ -204,6 +205,11 @@ def autosomes(self, also=()):
 
     def by_chromosome(self):
         """Iterate over bins grouped by chromosome name."""
+
+        # necessary because of pandas bug which says:
```

Comment:
So @etal 

I have found the "bug causing element" and reported the following bug here:  pydata/pandas#13179